redis.conf

```diff
@@ -860,10 +860,11 @@ replica-priority 100
 #               The special category @all means all the commands, but currently
 #               present in the server, and that will be loaded in the future
 #               via modules.
-#  +<command>|first-arg    Allow a specific first argument of an otherwise
-#                          disabled command. Note that this form is not
-#                          allowed as negative like -SELECT|1, but
-#                          only additive starting with "+".
+#  +<command>|first-arg  Allow a specific first argument of an otherwise
+#                        disabled command. It is only supported on commands with
+#                        no sub-commands, and is not allowed as negative form
+#                        like -SELECT|1, only additive starting with "+". This
+#                        feature is deprecated and may be removed in the future.
```

Comment:
```suggestion
#                        feature is deprecated in Redis 7.0 and may be removed
#                        in the future.
```

Reply:
why does it matter which release it was deprecated in?
if something is deprecated people are just advised to avoid using it.

it's not that 7.0 added an alternative, which means that starting 7 you should use that instead of this.
it's that this feature of using it on non-sub-commands was born in a sin, and should never be used.

the other part of it, which is using it on sub-commands, was limited till now, and 7.0 improves it.

Reply:
It's a hint about what "in the future" refers too. If we never actually remove anything, it doesn't matter though.

In some projects, it's fine to remove stuff that has been deprecated 2 or 3 major releases back, so if any users assume that about Redis, it is useful to know since when it's deprecated.

Reply:
we do remove things from time to time (in 7.0 we did remove quite a few things).

i'm just arguing that in some cases, version 1 includes an API, then 2 can introduce an alternative API and mark the first as deprecated, so that version 3 can completely remove the first one.
but in this case, it's not a feature that was replaced by an alternative (an alternative which only exists in 7), it's something we never meant to be used, and we advise to never use it.

Reply:
Fine. Now it has at least been considered. :-)